spec/features/orders/user_cancels_order_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe "As a registered user" do
   describe "when I visit an orders show page" do
-    xit "I click cancel order button" do
+    it "I click 'Cancel Order' button only if order is pending" do
```

Comment:
```suggestion
    xit "I click 'Cancel Order' button only if order is pending" do
```

I'm going to go ahead and suggest that we keep this x'd out since we will fail the test until we come back to user story 27. Other than that, love everything in this pull request!